l3backend/l3backend-color.dtx

```diff
@@ -1248,6 +1253,401 @@
 %</package>
 %    \end{macrocode}
 %
+% \subsection{Color in \LuaTeX{}}
+%
+%    \begin{macrocode}
+%<*lua>
+%    \end{macrocode}
+%
+% In \LuaTeX{}, we can use an approach based on attributes to avoid having to
+% insert whatsits. That means having the appropriate supporting Lua code. The
+% underlying idea comes from Heiko Oberdiek's \pkg{luacolor} package.
+%
+% \begin{macro}{ltx.color}
+% \begin{macro}{ltxcolor}
+%   The basic table should be set up in \pkg{l3luatex}, but we want a separate
+%   sub-table for color.
+%    \begin{macrocode}
+ltx = ltx or {color = {}}
+ltx.color = ltx.color or {}
+local ltxcolor = ltx.color
+%    \end{macrocode}
+% \end{macro}
+% \end{macro}
+%
+% Load basic support and print some information.
+%    \begin{macrocode}
+require("ltluatex")
+local luatexbase = luatexbase
+luatexbase.provides_module({name = "l3backend-luatex", date = "2021-06-03"})
+%    \end{macrocode}
+%
+% \begin{macro}{attr}
+%   An attribute to track color. There is no need to access this from \TeX{} so
+%   in contrast to \pkg{luacolor} we just allocate directly in Lua.
+%    \begin{macrocode}
+local color_attr = luatexbase.new_attribute("color")
+%    \end{macrocode}
+% \end{macro}
+%
+% \begin{macro}{map}
+%   A map between color strings and their color_index, plus the tracking number.
+%    \begin{macrocode}
+local map = {n = 0}
+%    \end{macrocode}
+% \end{macro}
+%
+% \begin{macro}{color_index}
+%   Get the color_index of a color or create a new color_index if necessary.
+%    \begin{macrocode}
+local function color_index(color)
+  local n = map[color]
+  if not n then
+    n = map.n + 1
+    map[n] = color
+    map[color] = n
+  end
+  return n
+end
+%    \end{macrocode}
+% \end{macro}
+%
+%   Local copies of global tables.
+%    \begin{macrocode}
+local node   = node
+local string = string
+local tex    = tex
+local texio  = texio
```

Comment:
`texio` isn't used.

Reply:
Fixed in 373c6090b930c0d9c1585c63c84995ccf5448870